internal/pkg/admission/scheduler/helper_test.go

```diff
@@ -0,0 +1,139 @@
+package scheduler
```

Comment:
This file already exists here https://github.com/storageos/cluster-operator/blob/master/internal/pkg/admission/scheduler/handler_test.go.
Why is it shown as a new file here?

Reply:
oops, never mind :sweat_smile: